src/components/CCPDashboard/ScanPatientButton.tsx

```diff
@@ -19,12 +20,16 @@ const useStyles = makeStyles({
 
 export const ScanPatientButton = () => {
```

Comment:
I am not familiar with best practices, but I would assume that a button that links to another page is quite common. Is it worth pulling out into `common` components instead?